assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -141,6 +234,32 @@ const baseSelectors = {
 		return state.dismissedTours;
 	},
 
+	/**
+	 * Gets a list of tour objects that qualify for the given view context.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state       Data store's state.
+	 * @param {string} viewContext View context.
+	 * @return {(Object[]|undefined)} Array of qualifying tour objects
+	 *                                `undefined` while readiness is being resolved.
+	 */
+	getReadyTours( state, viewContext ) {
```

Comment:
Let's name this `getCurrentTours` or, I think better, `getCurrentFeatureTours`, similar to how it was discussed during IB review. If we rename this to `getCurrentFeatureTours`, I'd also suggest to rename the above selector to `getDismissedFeatureTours`.

Reply:
Not a problem to rename, but I feel like "current" doesn't quite have the same meaning that the returned tours are "qualified" and thus ready to show. I think an added `ForView` would also be a bit more clear.